.travis.yml

```diff
@@ -8,13 +8,13 @@ addons:
     - docker
     - chromium
 
-before_install:
+install:
   - yarn global add wait-on
   # Install Codecov
   - yarn install
   - cp cypress.env.template.json cypress.env.json
 
-install:
+before_script:
```

Comment:
how is this and the change :point_up_2: related to this `PR`? 
is this some setup that should be changed anyway, ie code creep, or is this somehow related to running the cucumber tests?

Reply:
Code creep, but a mild one. Starting the docker containers is more a `before_script` whereas installing e.g. `cucumber` is part of the `install` in my eyes.